local_planner/src/nodes/local_planner.cpp

```diff
@@ -216,42 +216,20 @@ void LocalPlanner::determineStrategy() {
 
       create2DObstacleRepresentation(send_obstacles_fcu_);
 
-      // check histogram relevance
-      bool hist_relevant = true;
-      if (!hist_is_empty_) {
-        hist_relevant = false;
-        int relevance_margin_z_cells =
-            std::ceil(relevance_margin_z_degree_ / ALPHA_RES);
-        int relevance_margin_e_cells =
-            std::ceil(relevance_margin_e_degree_ / ALPHA_RES);
-        int n_occupied_cells = 0;
-
-        PolarPoint goal_pol = cartesianToPolar(goal_, position_);
-        Eigen::Vector2i goal_index = polarToHistogramIndex(goal_pol, ALPHA_RES);
-
-        for (int e = goal_index.y() - relevance_margin_e_cells;
-             e < goal_index.y() + relevance_margin_e_cells; e++) {
-          for (int z = goal_index.x() - relevance_margin_z_cells;
-               z < goal_index.x() + relevance_margin_z_cells; z++) {
-            if (polar_histogram_.get_dist(e, z) > FLT_MIN) {
-              n_occupied_cells++;
-            }
-          }
-        }
-        if (n_occupied_cells > 0) {
-          hist_relevant = true;
-        }
-      }
-
       // decide how to proceed
-      if (hist_is_empty_ || !hist_relevant) {
+      if (hist_is_empty_) {
         obstacle_ = false;
         waypoint_type_ = tryPath;
       }
 
-      if (!hist_is_empty_ && hist_relevant && reach_altitude_) {
+      if (!hist_is_empty_ && reach_altitude_) {
         obstacle_ = true;
 
+        float yaw_angle = std::round((-curr_yaw_ * 180.0f / M_PI_F)) + 90.0f;
```

Comment:
Why do we add 90?

Reply:
because the histogram angle zero starts at the y-axis, wheras the curr_yaw comes from the firmware and is x-axis oriented